modules/channel-web/src/views/lite/components/messages/MessageGroup.tsx

```diff
@@ -40,7 +40,7 @@ const MessageGroup = (props: MessageGroupProps) => {
               payload.url = (data.message_data && data.message_data.url) || (data.message_raw && data.message_raw.url)
             }
 
-            if (props.messageWrapper && payload.type !== 'session_reset') {
```

Comment:
It is important not to wrap the message if it's a session reset.

Reply:
Good point